NVActivityIndicatorView/NVActivityIndicatorView.swift

```diff
@@ -75,54 +138,17 @@ public class NVActivityIndicatorView: UIView {
         Create a activity indicator view with specified type, color, size and size
         
         :param: frame view's frame
-        :param: type animation type, value of NVActivityIndicatorType enum
-        :param: color color of activity indicator view
-        :param: size actual size of animation in view
+        :param: type animation type, value of NVActivityIndicatorType enum. Default type is pacman.
+        :param: color color of activity indicator view. Default color is white.
+        :param: size actual size of animation in view. Default size is 40
     
         :returns: The activity indicator view
     */
-    public init(frame: CGRect, type: NVActivityIndicatorType, color: UIColor?, size: CGSize?) {
+    public init(frame: CGRect, type: NVActivityIndicatorType = DEFAULT_TYPE, color: UIColor = DEFAULT_COLOR, size: CGSize = DEFAULT_SIZE) {
```

Comment:
Should we still need optional here? If passing nil params like this `NVActivityIndicatorView(frame: <value>, type: <value>, color: nil, size: <value>`, this will cause error. Any suggestion?